### PR TITLE
Update: n.Makefile and lintspaces-cli version

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy @andygnewman @bjfletcher

### CODE REVIEW

File: server/lib/load-list.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import api from 'next-ft-api-client';
 import {coroutine} from 'bluebird';
 
 /**
- * Downloads a list of articles from CAPI or Bertha and returns the UUIDs, plus any metadata.
- */
+* Downloads a list of articles from CAPI or Bertha and returns the UUIDs, plus any metadata.
+*/
```
Comment (ironsidevsquincy) [inline server/lib/load-list.js]: hmmmm, these should be ok...what's the error?

Comment (andygout) [inline server/lib/load-list.js]: `Unexpected spaces found. [warning]`

Comment (ironsidevsquincy) [inline server/lib/load-list.js]: That's a editor config error i think (rather than eslint). Make sure your node modules are up to date

Comment (ironsidevsquincy) [inline server/lib/load-list.js]: Actually, think you'll need to update `lintspaces-cli` to the latest version (`0.4.0`)

File: test/server/transforms/special-reports-hub-featured.test.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/*eslint no-console: 0*/
```
Comment (ironsidevsquincy) [inline test/server/transforms/special-reports-hub-featured.test.js]: does this test file need a console log?

**Pull request merged**